### PR TITLE
Update storage-connection-strings.md

### PR DESCRIPTION
**Pull request by anshulsharmas**
Added support for S3 presigned url

### CODE REVIEW

**Comment by Court72** [comment]
@shsagir

Can you review the proposed changes? Note that the [validation warning](https://github.com/MicrosoftDocs/dataexplorer-docs/pull/2105#issuecomment-1873643147) must be resolved before merging.

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: data-explorer/kusto/api/connection-strings/storage-connection-strings.md [hunk 1]
```diff
@@ -127,3 +129,10 @@ To add Amazon Web Services access keys, append `;AwsCredentials={ACCESS_KEY_ID},
 |Example|
 |--|
 |`"https://yourbucketname.s3.us-east-1.amazonaws.com/path/to/file.csv;AwsCredentials=AWS1234567890EXAMPLE,1234567890abc/1234567/12345678EXAMPLEKEY"`|
+
+### Amazon Web Services S3 presigned URL
+
+Use the [S3 presigned URL](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ShareObjectPreSignedURL.html) as the connection string.
+|Example|
```
Comment (v-dkorn) [inline data-explorer/kusto/api/connection-strings/storage-connection-strings.md:136]:
```suggestion

|Example|
```

**Comment by shsagir** [comment]
#sign-off

File: data-explorer/kusto/api/connection-strings/storage-connection-strings.md [hunk 2]
```diff
@@ -62,6 +63,7 @@ The following table summarizes the available authentication methods for differen
 | [Microsoft Entra access token](#azure-ad-access-token) | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :x: | Microsoft Entra tokens have an expiration time. Use when accessing storage for a limited time. |
 | [Storage account access key](#storage-account-access-key) | :heavy_check_mark: | :heavy_check_mark: | :x: | :x: | When you need to access resources on an ongoing basis. |
 | [Amazon Web Services Programmatic Access Keys](#amazon-web-services-programmatic-access-keys) | :x: | :x: | :x: | :heavy_check_mark: | When you need to access Amazon S3 resources on an ongoing basis. |
+| [Amazon Web Services S3 presigned URL](#amazon-web-services-s3-presigned-url) | :x: | :x: | :x: | :heavy_check_mark: | When you need to access Amazon S3 resources with a temp presigned url. |
```
Comment (Court72) [inline data-explorer/kusto/api/connection-strings/storage-connection-strings.md:66]:
```suggestion
| [Amazon Web Services S3 presigned URL](#amazon-web-services-s3-presigned-url) | :x: | :x: | :x: | :heavy_check_mark: | When you need to access Amazon S3 resources with a temp presigned URL. |
```

**Pull request merged**